package-lock.json

```diff
@@ -59045,7 +59045,7 @@
         "lodash": "^4.17.21",
         "mocha": "^10.2.0",
         "mongodb": "^6.5.0",
-        "mongodb-data-service": "*",
```

Comment:
This should be fine right?

Reply:
is it missing some rebase, or just being generally out of sync? i don't get why this happens to be honest, it changed mongodb-data-service in other places too, but i don't see a reason why we would want that.

Reply:
I see it went away after recreating, but generally sometimes it happens, when you do `npm i --save <package> -w <workspace>` in some cases npm first adds it to package-lock with a `*` but with a correct version to package.json, and only on subsequent install it updates the package-lock